activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -118,76 +118,76 @@ def test_unscope_after_reordering_and_combining
   end
 
   def test_unscope_with_where_attributes
-    expected = Developer.order("salary DESC").collect(&:name)
-    received = DeveloperOrderedBySalary.where(name: "David").unscope(where: :name).collect(&:name)
+    expected = Developer.order("salary DESC").order(:name).collect(&:name)
```

Comment:
Do you remember why we need to add a new order here?

Reply:
The build was resulting in a different order between expected and received, I normalized both by using a predictable order.

Not sure why it started to fail in this PR but I have assumed that Postgres changed the order of the results for some reason (there is no guarantee in the order of the results in Postgres unless specified, in that case, if the salary is the same we do not have a guaranteed order of the records [there is but it is the order of the record on the disk that may vary from time to time]).

In the other hand, I'm not sure if this is the real problem but since this PR change should not impact on this test at all I assumed it was that.

Reply:
might be better to do `assert_equal(expected.sort, received.sort)`

Reply:
Yeah, we've used `sort` and `sort_by` for that. 75cfb3dbdbb520147a532bd92ad6111c0e2ebf88 93c65ae4efd62933618193c088ed1dadef47793d 625bdb164ed7cb06dd88bec57c338b63460628d7

Reply:
Pretty much the same, should I change it to sort in ruby instead of the database?

Reply:
I prefer to because of the already existing `order` that we have on those tests